org/responses.py

```diff
@@ -23,6 +23,35 @@
   "message": "Already a member of organization"
 }
 
+create_group_400={
+  "name": [
+    "Ensure this field has no more than 30 characters."
+  ],
+  "role": [
+    "Ensure this field has no more than 200 characters."
+  ],
+  "permissions_array": [
+    "This list may not be empty.",
+    "Ensure this value is less than or equal to 4.",
+    "Ensure this value is greater than or equal to 0."
```

Comment:
```suggestion
```

 `"This list may not be empty."` - Volunteer grp has no permissions, so making sure that the list is not null will be enough


`For "Ensure this value is less than or equal to 4." & "Ensure this value is greater than or equal to 0.":` 
Adding new permissions will make this invalid. So instead remove the above error. Instead when the `custom_model_field.get_permission_list()` doesn't have the entered permission int, provide `The permission ints are not valid` error msg.

 A good way to implement this would be to verify the permission_ints inside the  `custom_model_field.set_permissions()`. This can be done with the help of `custom_model_field.get_permission_list()`. If the perm int is not found then raise a exception `PERMISSION_INT_INVALID` and catch that in the view function or serailizer where appropriate!

Reply:
Made these changes in the latest commit :v: